app/src/main/res/layout/activity_main.xml

```diff
@@ -17,6 +17,15 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recyclerView"
+            android:layout_width="413dp"
+            android:layout_height="670dp"
```

Comment:
I think we can use `match_parent` for width and height instead of hard coding it.